src/RigidBodyTreeInspector.jl

```diff
@@ -152,4 +153,20 @@ function inspect(mechanism; show_inertias::Bool=false, randomize_colors::Bool=tr
         end, [Interact.signal(w) for w in widgets]...)
 end
 
+import Base: one
+one(::Type{Array{Float64,1}}) = 1. # FIXME: this is a hack to get gridded interpolation from Interpolations to work on vectors
```

Comment:
Any thoughts on this? Had to do this because at https://github.com/tlycken/Interpolations.jl/blob/master/src/gridded/gridded.jl#L31, Interpolations uses the `one` function on the element type of the vector input (which is `Vector{Float64}` in my case). Should I do a pull request against Interpolations to fix it, or is there a better way?